src/_data/people.json

```diff
@@ -0,0 +1,69 @@
+{
+	"Michael Chan": { "github": "chantastic" },
```

Comment:
How would we feel about keeping this list alphabetized? I think that'd make it easier to verify whether someone's in the list already, or to make updates.

Reply:
done!